docs/platform/feature-flags/stale-feature-notifications.md

```diff
@@ -0,0 +1,86 @@
+---
+title: BETA - Stale Feature Notifications 
+sidebar_position: 5
+---
+
+:::info
+Stale Feature Notifications are a Premium feature. To learn more, read about our [pricing](https://devcycle.com/pricing). To upgrade your plan, please update your plan on your Billing Information page which can be found in your organization's Settings, or contact [Sales](mailto:sales@devcycle.com).  
+:::
+
+Stale Features are identified based on specific conditions associated with their usage, modifications, and Feature type. DevCycle will alert you when a Feature has been qualified as potentially stale to help ensure that Features are surfaced for cleanup. 
+
+If a Feature is marked as stale, DevCycle encourages users to take action by updating its status to Complete or Archived. This indicates that a Feature is ready to be cleaned up and removed from your codebase. 
+
+## Stale Feature Reasons 
+
+Features can belong to one of the following staleness reaons:
+
+### Unmodified - Short-Lived 
+
+A Feature is classified as `Unmodified - Short-Lived` if it has not been updated for more than 1 month.
+
+**Applicable to**: Release, Experiment Feature types.
+
+### Unmodified - Long-Lived 
```

Comment:
should we merge both `Unmodified` into a single section and have sub-headers by Feature Type instead?

Reply:
also, the timelines stated here are the same, just in different units.

short-lived is stated as unmodified for 1 month and long-lived for 30 days.